### PR TITLE
fix(notification-drawer): updated text on demos

### LINKED ISSUE #3908: Update read/unread text in notification drawer demo
In the read notification drawer example, some of the notifications say "unread."

https://www.patternfly.org/v4/components/notification-drawer/html-demos/expanded-read/

Screen Shot 2021-03-03 at 5 14 17 PM

### PR DESCRIPTION
**Pull request by bitwiseor**
fixes #3908 

also updated the danger notification show it only shows on the "Expanded attention" demo; the other demos have an extra recommendation notification instead, to maintain a consistent number of notifications across demos

### CODE REVIEW

File: src/patternfly/components/ContextSelector/context-selector-toggle.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-context-selector__toggle{{#if context-selector-toggle--modifier}} {{context-selector-toggle--modifier}}{{/if}}"
+<button class="pf-c-context-selector__toggle{{#if context-selector-toggle--IsPlain}} pf-m-plain{{/if}}{{#if context-selector-toggle--modifier}} {{context-selector-toggle--modifier}}{{/if}}"
```
Comment (mcoker) [inline src/patternfly/components/ContextSelector/context-selector-toggle.hbs:1]: You have some context selector changes in this PR. It looks like maybe you created this branch from the context selector branch you were working on? You can see this branch has 2 commits - one for the context selector, and one for the notification drawer work - https://github.com/patternfly/patternfly/pull/4117/commits

File: src/patternfly/demos/NotificationDrawer/examples/NotificationDrawer.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ section: components
 
 ### Expanded attention
 ```hbs isFullscreen
-{{> notification-drawer-demo id="drawer-expanded-attention" page-template-header-tools-notification-badge--badge--modifier="pf-m-attention" page-header-tools--IsAttention="true" drawer-panel--IsOpen="true"}}
+{{> notification-drawer-demo id="drawer-expanded-attention" page-template-header-tools-notification-badge--badge--modifier="pf-m-attention" page-header-tools--IsAttention="true" drawer-panel--IsOpen="true" notification-drawer-basic-list--IsAttention="true"}}
 ```
```
Comment (mcoker) [inline src/patternfly/demos/NotificationDrawer/examples/NotificationDrawer.md:26]: Nice!!

**Review by mcoker — CHANGES_REQUESTED** [review]
Great job on this!! We just need to make sure this is a branch off of master and doesn't include the context selector work.

**Review by mcarrano — CHANGES_REQUESTED** [review]
@bitwiseor Thanks for making these changes. The text in the notifications now matches the visual state, however the Unread count in the header is not correct. For example, they all say "2 unread" when in some cases there are 3 or none. Can you update that text to make sure it's consistent with the number of unread in the example?

File: src/patternfly/components/ContextSelector/context-selector-toggle.hbs (hunk 1 above)
Comment (bitwiseor) [reply src/patternfly/components/ContextSelector/context-selector-toggle.hbs]: Fixed in https://github.com/patternfly/patternfly/pull/4117/commits/93f0792c68c556a4743398c3e0538e53de51bf8b

**Comment by bitwiseor** [comment]
Context selector work in this branch has been reversed and notification drawer demos have been updated to reflect the correct number of unread notifications. I wasn't sure about the groups demos; I said 9 unread notifications, since there were 3 in the expanded group and 2 each in the other 3 groups.

**Review by mcarrano — APPROVED** [review]
Looks great @bitwiseor . Thanks for making these updates.

File: src/patternfly/demos/NotificationDrawer/notification-drawer-demo-basic.hbs [hunk 3]
```diff
@@ -4,7 +4,11 @@
       Notifications
     {{/notification-drawer-header-title}}
     {{#> notification-drawer-header-status}}
-      2 unread
+      {{#if notification-drawer-basic-list--AllRead}}
```
Comment (mcoker) [inline src/patternfly/demos/NotificationDrawer/notification-drawer-demo-basic.hbs:7]: nice!

**Review by mcoker — APPROVED** [review]
great work!! 🥳

**Pull request merged**